origin.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>  
+        <header class="navbar">
+            <h3>Hector Andre Lau</h3>
+            <img src="./images/Icon-Menumenuicon.svg" alt="Menu">
+        </header>    
+        
+            <h1>Hey there, I'm Hector Andre Lau</h1>
+            <h2>I'm a software developer with experience in finance and logistic processes.</h2>
+
+        <p>I can build a website meeting the expected requirements, pluss I can add my actual knowledge of how the process need to work on certain areas like logistic, account, finance and billing and all of this focus to the good experience and satisfaction ot the customer.</p>
```

Comment:
- [ ] Kindly add the margins (top, bottom, left, right) defined in the figma design
- [ ] Please use font colors specified in the design
 Check the screenshots  below 🤗 
### Figma Design

![Untitled design (1)](https://user-images.githubusercontent.com/69638013/153395423-0e6be8fc-89d9-4c97-9169-2a8e02a40523.png)
### Your design
![aab](https://user-images.githubusercontent.com/69638013/153396140-1ca9dd4e-deae-4931-8a04-c396fa525917.png)